crates/cheatnet/tests/cheatcodes/spoof.rs

```diff
@@ -0,0 +1,309 @@
+use crate::{
+    assert_success,
+    common::{deploy_contract, get_contracts, recover_data, state::create_cheatnet_state},
+};
+use cairo_felt::Felt252;
+use cheatnet::{
+    conversions::{
+        class_hash_to_felt, contract_address_to_felt, felt_from_short_string,
+        felt_selector_from_name,
+    },
+    rpc::call_contract,
+};
+
+#[test]
+fn spoof_simple() {
```

Comment:
We should also verify that if spoofing a one argument, other remain unchanged (not spoofed).

Reply:
Preferably in a different test IMO